src/fastcs/__init__.py

```diff
@@ -7,5 +7,7 @@
 """
 
 from ._version import __version__
+from .launch import FastCS as FastCS
+from .launch import launch as launch
```

Comment:
Why do we need the `as`?

Reply:
Good question. My understanding is a bit bottom up on this. Basically if I remove the `as` then you will get ruff: `src/fastcs/__init__.py:11:21: F401 .launch.launch imported but unused; consider removing, adding to '__all__', or using a redundant alias.` To solve this I looked over at https://github.com/fastapi/fastapi/blob/master/fastapi/__init__.py and this is how they do it. I dont have a PEP to give you

Reply:
Ah it does the same as `__all__`, but neater. OK sure.

Reply:
Ah yes you are right. Even better. It allows it to be imported from the top-level explicitly (without unused import errors, which is the second effect of `__all__`), but without allowing to be imported as with `*`